moulinette/authenticators/ldap.py

```diff
@@ -59,6 +59,8 @@ def __del__(self):
 
     @property
     def is_authenticated(self):
+        if not self.con:
```

Comment:
The code should be:

```suggestion
        if self.con is None:
```

As travis is complaining about (do you read thoses?)